### PR TITLE
Homework 4

### PR DESCRIPTION
**Pull request by iliaaa**

### CODE REVIEW

File: src/test/java/hw/four/AbstractSettingClass.java [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+
+public abstract class AbstractSettingClass {
+
+    public SingletonForProperties singletonForProperties = SingletonForProperties.getInstance();
+    public WebDriver driver;
+
+    ActionStep actionStep;
+    AssertionStep assertionStep;
```
Comment (akulebyakin) [inline src/test/java/hw/four/AbstractSettingClass.java]: These class fields can be protected not public

**Review by akulebyakin — CHANGES_REQUESTED** [review]
Please resolve CheckStyle warnings
image

File: src/test/java/hw/four/AbstractSettingClass.java (hunk 1 above)
Comment (iliaaa) [reply src/test/java/hw/four/AbstractSettingClass.java]: changed

**Review by akulebyakin — APPROVED** [review]
Good job!

**Pull request merged**